doc/ar65.sgml

```diff
@@ -81,13 +82,13 @@ This will delete the module named `sub1.o' from the library, printing an
 error if the library does not contain that module.
 
 
-The `l' command prints a list of all modules in the library. Any module
-names on the command line are ignored.
+The `t' command prints a table of all modules in the library (`l' is deprecated).
```

Comment:
Same here: I would expect the 'l' to be mentioned in the list of operations, too, with a remark that it is deprecated.